### PR TITLE
Update site for Beta launch

### PR DESCRIPTION
**Pull request by simonrjones**

### CODE REVIEW

**Comment by simonrjones** [comment]
I’ve added a [new draft post](https://deploy-preview-113--ecstatic-mccarthy-0be5bd.netlify.app/updates/beta-site-launch) for the WiO site to publish on Monday morning.

And updated a few other key pages:
* [Homepage](https://deploy-preview-113--ecstatic-mccarthy-0be5bd.netlify.app/)
* [Feedback page](https://deploy-preview-113--ecstatic-mccarthy-0be5bd.netlify.app/feedback/) (I added Mastodon links, hope that’s OK. Had to put mine for S24 since we are not currently checking Mastodon on our agency account yet)

If there is any W3C blog planned I can add a link to that too on the day.

File: _posts/2023-02-27-beta-site-launch.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+The new design for W3C has been launched on the Beta website [beta.w3.org](https://beta.w3.org/) to gather community feedback. 
+
+This is an exciting development and is the result of over 2 years of work from [W3C](https://www.w3.org/) and [Studio 24](https://www.studio24.net/), 
+with help from [Craft CMS](https://craftcms.com/), [Digital Accessibility Centre](https://digitalaccessibilitycentre.org/) and 
+[Zoonou](https://zoonou.com/).
+
+Studio 24 worked on the project between March 2020 and April 2022 and have provided support to W3C since then. The W3C 
+Systems Team, Marketing and Communications Team, and others have been hard at work building pages using the design system, 
+writing content and updating the Beta website.
```
Comment (vivienlacourba) [inline _posts/2023-02-27-beta-site-launch.md:20]: I think you can add importing legacy content to that list.

**Review by vivienlacourba — APPROVED** [review]

**Comment by simonrjones** [comment]
@vivienlacourba updated: https://deploy-preview-113--ecstatic-mccarthy-0be5bd.netlify.app/updates/beta-site-launch

Thanks for approving, I'll merge and deploy Monday morning.

**Comment by koalie** [comment]
(I've removed myself from the "reviewers"; I don't know how to "approve" here but did in Slack; sorry!)

**Pull request merged**